src/os/bluestore/BlueStore.cc

```diff
@@ -6684,15 +6684,19 @@ int BlueStore::_read_bdev_label(
     decode(expected_crc, p);
   }
   catch (ceph::buffer::error& e) {
-    derr << __func__ << " unable to decode label " << path.c_str()
-         << " at offset " << p.get_off()
-	 << ": " << e.what()
-	 << dendl;
+    // We can still get here in non-erroneous scenarios,
```

Comment:
This will collide with https://github.com/ceph/ceph/pull/61470/files#diff-6f3b1a5dcf5313dc06f71ba4997ee28ec219ecb5633a728b916465c684f1d87dR6687-R6688 .
But this was first, so I will resolve conflict after this merges.